### PR TITLE
added UserButton Component + token in headers if not null

### LINKED ISSUE #15: <UserButton>

### PR DESCRIPTION
**Pull request by omthakare16**
this P.R fixes #15 and #17 issues.

### CODE REVIEW

File: src/components/UserButton/UserButton.jsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+          <div
+            style={{
+              display: 'flex',
+              alignItems: 'center',
+              marginBottom: '10px',
+              overflow: 'hidden',
+              textOverflow: 'ellipsis',
+            }}
+          >
+            <img
```
Comment (ghoshbishakh) [inline src/components/UserButton/UserButton.jsx:90]: Can this be made a separate file from where it is imported? I am not sure though.

Comment (omthakare16) [reply src/components/UserButton/UserButton.jsx:90]: yes, it can be returned by some function from a separate file.
should I do it that way?

Comment (omthakare16) [reply src/components/UserButton/UserButton.jsx]: added in commit a106b57264b22dbdcb6410e843a0a7d4067cd611

**Comment by omthakare16** [comment]
a106b57264b22dbdcb6410e843a0a7d4067cd611 commit fixes issue #16

**Comment by ghoshbishakh** [comment]
Thanks @omthakare16

**Pull request merged**